includes/functions/llms.functions.templates.dashboard.php

```diff
@@ -153,7 +153,7 @@ function lifterlms_student_dashboard( $options = array() ) {
 }
 
 
-if ( ! function_exists( 'lifterlms_template_my_courses_loop' ) ) {
+if ( ! function_exists( 'llms_template_my_courses_loop' ) ) {
```

Comment:
This is an oversight, please revert this change, we don't want to change this function name, we only want that new functions added here respect the new naming convention. Please revert these changes, you only need to change the prefix of the new functions you added.

Reply:
fixed now. my bad.